rep-0003.rst

```diff
@@ -220,18 +216,64 @@ Exact or Series Requirements:
 - Qt 5.3.x
 - PyQt5
 
-Buildsystem support:
+Build System Support:
 
-- catkin:
+- Same as Indigo
 
-  - build from source
-  - release for binary packaging
-  - wiki documentation
-  - continuous integration
+Lunar Loggerhead (May 2017 - May 2019)
+--------------------------------------
+Required Support for:
 
-- rosbuild:
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
 
-  - build from source
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 26
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Minimum Requirements:
+
+ Most of the following requirements are the same as Kinetic's to avoid maintainers
+ branching out before the next LTS release
+
+- C++11
+- Python 2.7
+
+  - Python 3.5 not required, but testing against it is recommended
+
+- Lisp SBCL 1.2.4
+- CMake 3.0.2
+- Boost 1.55
+
+Exact or Series Requirements:
+
+- Ogre3D 1.9.x
+- Gazebo 8 (packaged by OSRF)
+
+  - ROS won't use upstream Gazebo for the upcoming few releases because of a mismatch
+    between upstream versions on targeted Ubuntu distributions
+  - In the case of Lunar:
+
+    - Zesty ships with Gazebo 8
+    - Yakkety and Xenial ship with Gazebo 7
+
+- PCL 1.7.x (Xenial)
+- PCL 1.8.x (Other Systems)
```

Comment:
>>    Packaging PCL1.8 would allow devs to target PCL1.8 specifically and thus require them to branch out from kinetic to release on lunar.

> I don't understand this bit, could you explain why "allow[ing] devs to target PCL1.8 specifically" is undesirable?

The indesirable effect that I'm seeing is the requirement to branch out, not the ability to use new features. There is also the side effect of not building against the system one as @jspricke pointed out.

>>    Otherwise we will need to make sure 1.8 is packaged for all kinetic supported platforms as well.

> Am I misunderstanding the "PCL 1.8.x (Other Systems)" line above, or would "other supported platforms" already have/need PCL 1.8?

Sorry if it was unclear. I was referring to all **kinetic** supported platforms, meaning platforms for which we build kinetic debian packages: Ubuntu Wily, Ubuntu Xenial and Debian Jessie that all ship with PCL1.7.